javascript-arrays/main.js

```diff
@@ -0,0 +1,25 @@
+var colors;
+colors = ['red', 'white', 'blue'];
+
```

Comment:
There is no need to do this on multiple lines, you can do this all on one line.

Reply:
Understood. I was using the method that Duckett used in the array introduction in the book. I will change it right now.